homeassistant/components/zwave_js/__init__.py

```diff
@@ -251,21 +258,30 @@ def async_on_notification(notification: Notification) -> None:
             },
         )
 
+    entry_hass_data: dict = {}
+    hass.data[DOMAIN][entry.entry_id] = entry_hass_data
```

Comment:
Based on line 268 it looks like you're assuming this data remains around. 
```suggestion
    entry_hass_data: dict = hass.data[DOMAIN].setdefault(entry.entry_id, {})
```